### DISCUSSION TITLE
Accessibility Issues on the Github Universe Site

### DISCUSSION THREAD

**Discussion by AccessiT3ch** [community/events#53] [Keynotes]
This is a live discussion providing a11y (accessibility) feedback to the fabulous GitHub crew! 

See / hear / feel something? Say something!

**Comment by AccessiT3ch** [comment]
As a sighted user, I should be able to see content displayed at color contrast ratio of at least 4.5:1, preferably 7:1, especially for body text and critical links.

Below, the session lineup link [on the schedule page](https://watch.githubuniverse.com/schedule) is rendered a dark blue on a dark (and busy) background:
 
<img width="349" alt="Screenshot 2022-11-09 at 1 05 40 PM" src="https://user-images.githubusercontent.com/95449364/200951656-254b121b-115b-4dea-a6b1-f0d850ef5fec.png">

**Reply by AccessiT3ch** [reply]
It might be helpful to include a text shadow on the link (if not the whole paragraph) to guarantee acceptable color contrast between the text and the dynamic background.

**Reply by AccessiT3ch** [reply]
Thanks for the quick fix! 🚀

**Comment by AccessiT3ch** [comment]
As a keyboard and mouse user, I should be able to access the schedule content.

1. I should see an indication that the page focus is in a horizontally scrolling element, so I know I can use my left / right arrow keys to access the horizontal overflow (of concurrent sessions).
2. I should be able to tab into the left/right buttons provided at the top of the schedule
3. I should be able to tab into something giving context to "Learning More +" and "Watch" in the session cards

<img width="1236" alt="Screenshot 2022-11-09 at 2 14 39 PM" src="https://user-images.githubusercontent.com/95449364/200954164-91abc105-6d7e-44c9-8d99-57fadd4d397b.png">

**Reply by AccessiT3ch** [reply]
For number 2, these buttons should probably be `<button>`s, or at least `<a>`s, so they're focusable.

**Comment by jricciardi** [comment]
Thank you for spinning this up @AccessiT3ch! GitHub always strives to make sure we're doing the best we can on a11y, but we know we're not perfect yet. Happy to make sure feedback is seen for improvements next year - keep the feedback coming ❤️

**Reply by AccessiT3ch** [reply]
My pleasure!

**Comment by AccessiT3ch** [comment]
As a keyboard navigator, I should be able to tab to an interactive element ([no interactive divs, pls!](https://benmyers.dev/blog/clickable-divs/)) to trigger the dropdown menu in the global header.

<img width="380" alt="Screenshot 2022-11-09 at 3 00 11 PM" src="https://user-images.githubusercontent.com/95449364/200960715-2202de0b-aa90-4ea0-a4df-3230aead461c.png">

(link credit: @BenDMyers 🙇)

**Comment by siaudvytisViktoras** [comment]
Thanks a lot for a proper presentation. I got very good observations!

**Comment by ghostinhershell** [comment]
Thank you @AccessiT3ch for all of the great accessibility feedback! ❤️
